tests/safety/common.py

```diff
@@ -219,6 +226,33 @@ def test_steer_req_bit_frames(self):
       self._set_prev_torque(self.MAX_TORQUE)
       self.assertTrue(self._tx(self._torque_cmd_msg(self.MAX_TORQUE, steer_req=1)))
 
+  def test_steer_req_bit_multi_invalid(self):
+    """
+      For safety modes allowing multiple consecutive invalid frames, this ensures that once a valid frame
+      is sent after an invalid frame (even without sending the max number of allowed invalid frames),
+      all counters are reset.
+    """
+    if self.MIN_VALID_STEERING_FRAMES == 0:
+      raise unittest.SkipTest("Safety mode does not implement tolerance for steer request bit safety")
```

Comment:
can we just run this test with everything set to zero? if not, why not?

Reply:
It would work if they were set to 0, however most safety modes don't have safety around the steer request bit, and we assume it does with these tests

Reply:
yeah, let's add a comment. I'd like to do this in the future